src/openads/infrastructure/connector/appnexus/AppNexusErrorException.js

```diff
@@ -0,0 +1,7 @@
+export default class AppNexusErrorException extends Error {
```

Comment:
how an OpenAds client would know that the recevied Error is an AppNexusErrorException?
Shouldn't it be exported?

Reply:
I think the best solution is to export types that are necessary for the client, so I will modify the index and expose all necessary things :+1: